README.md

```diff
@@ -91,63 +90,51 @@ The configuration parameters are provided through environment variables:
+- **AllowedQueriesChecker**: There are URL requests that the server can allow
+  without going through an access checker.
+  [`AllowedQueriesChecker`](https://github.com/google/fhir-access-proxy/blob/main/server/src/main/java/com/google/fhir/proxy/AllowedQueriesChecker.java)
   is a special `AccessChecker` that compares the incoming request with a
-  configured set of allowed-queries. The intended use of this checker is to
-  override all other access-checkers for certain user-defined criteria. The user
-  defines their criteria in a config file and if the URL query matches an entry
-  in the config file, access is granted. An example of this is
-  [`hapi_page_url_allowed_queries.json`](https://github.com/google/fhir-gateway/blob/main/resources/hapi_page_url_allowed_queries.json).
-  To use this file with `ALLOWED_QUERIES_FILE`:
+  configured set of allowed-queries. 
+  
+  The intended use of this checker is to override all other access-checkers for certain user-defined criteria. The user defines their criteria in a config file and if the URL query matches an entry in the config file, access is granted. An example of this is:
+  [`hapi_page_url_allowed_queries.json`](https://github.com/google/fhir-access-proxy/blob/main/resources/hapi_page_url_allowed_queries.json).
+  To use the file, set the `ALLOWED_QUERIES_FILE` variable:
 
   ```shell
   export ALLOWED_QUERIES_FILE="resources/hapi_page_url_allowed_queries.json"
   ```
 
-- `BACKEND_TYPE`: The type of backend, either `HAPI` or `GCP`. `HAPI` should be
-  used for most FHIR servers, while `GCP` should be used for GCP FHIR stores.
-
-## Gateway to server access
-
-The proxy must be able to send FHIR queries to the FHIR server. The FHIR server
-must be configured to accept connections from the proxy while rejecting most
-other requests.
-
-If you use a
-[GCP FHIR store](https://cloud.google.com/healthcare-api/docs/concepts/fhir) you
-have the following options:
-
-- If you have access to the FHIR store, you can use your own credentials by
-  doing
-  [application-default login](https://cloud.google.com/sdk/gcloud/reference/auth/application-default/login).
-  This is useful when testing the proxy on your local machine, and you have
-  access to the FHIR server through your credentials.
-- Use a service account with required access (e.g., "Healthcare FHIR Resource
-  Reader", "Healthcare Dataset Viewer", "Healthcare FHIR Store Viewer"). You can
-  then run the proxy in the same GCP project on a VM with this service account.
-- [not-recommended] You can create and download a key file for the above service
-  account, then use it with
+- The FHIR Info Gateway makes no assumptions about what the FHIR server is, but it should be able to send any FHIR queries to the server. For example, if you use a [GCP FHIR store](https://cloud.google.com/healthcare-api/docs/concepts/fhir) you have the following options:
```

Comment:
Again it seems you are reverting changes in #83; please first sync to the current `main` and only apply the changes that are needed.